images/pidns.proto

```diff
@@ -0,0 +1,5 @@
+syntax = "proto2";
+
+message pidns_entry {
+	optional string ext_key		= 1;
```

Comment:
I was also thinking about this. Not sure. I was looking at the external network namespace support implementation and that uses optional. With PID namespace support it is however the only entry in the protobuf definition. Having the only field as optional is kind of strange.

So, yes, `required` would be probably a good idea right now as the file will only exist if `ext_key` is set. If we ever get another entry here then it would be better to be `optional`.

Interestingly I just read that the `proto3` format has dropped `required` and `optional` completely.

I will change it to `required` if we think that this is better. I am unsure myself what do to with it.